public/pages/admin.html

```diff
@@ -0,0 +1,47 @@
+<div ng-controller="AdminController">
+	<script src="./../javascripts/main.js"></script>
+	<style type="text/css">
+		#admin-table {
```

Comment:
Pfft css in an HTML doc. 
http://giphy.com/gifs/mind-half-amatuer-WGgMcIryybEgU